clean.sh

```diff
@@ -1,3 +1,2 @@
 #!/bin/bash
 rm -rf debian/build docker/build
-
```

Comment:
Allgemein ist empfohlen, Dateien mit einem newline zu beenden.
https://thoughtbot.com/blog/no-newline-at-end-of-file